### PR TITLE
Remove vim modelines from files

### PR DESCRIPTION
**Pull request by gauthierm**
Editorconfig handles this better.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -12,7 +12,7 @@ insert_final_newline = true
 indent_style = space
 indent_size = 2
 
-[*.{php,js,css,json,xml}]
+[*.{php,js,css,json,xml,rng}]
```
Comment (nburka) [inline .editorconfig]: Did you mean to include this?

**Comment by nburka** [comment]
R+ apart from my comment above

File: .editorconfig (hunk 1 above)
Comment (gauthierm) [inline .editorconfig]: It should be a separate PR
